src/ts/views/buttons/circle.tsx

```diff
@@ -9,14 +9,22 @@ const CircleButton = (props: {
   onClick?: () => void,
   disabled?: boolean
 }) => {
+  const {
+    className,
+    style,
+    onClick,
+    disabled,
+    children
+  } = this.props;
```

Comment:
this guy is a stateless component, it wont have `this.props`, just the `props` argument